assets/src/components/propertiesPanel/header.tsx

```diff
@@ -29,6 +29,7 @@ interface Props {
   vehicle: Vehicle | Ghost
   tabMode: TabMode
   setTabMode: Dispatch<SetStateAction<TabMode>>
+  closePanel: () => void
```

Comment:
nit(non-blocking): can this be changed to `onClosePanel` as well?
https://github.com/mbta/skate/pull/2263#discussion_r1367379895

Reply:
768a696c